scripts/build_automation/automateVersioning.sh

```diff
@@ -95,22 +95,9 @@ updateVersionAndBuildNumberInPlist() {
 
 generateReleaseNotes() {
 	TAG="$APP_NAME-$APP_RELEASE_VERSION"
-	#	if the tag exists, use the tag 2nd to last in the list, else use the last tag in the list
-	NUMBER_FROM_THE_BOTTOM=1
-	if git ls-remote --tags | egrep -q "refs/tags/$TAG$"
-	then
-		NUMBER_FROM_THE_BOTTOM=2
-	fi
-	
-	
-	PREVIOUS_RELEASE_TAG=$(git ls-remote --tags origin | grep $APP_NAME- | sort -V | tail -$NUMBER_FROM_THE_BOTTOM | head -n 1 | awk '{print $2}' | sed -e "s/refs\/tags\///g")
```

Comment:
the list of tags includes the hash first, so the `sort -V` was I think sorting by the hashes. I could have kept fixing here, but I like the generate-release-notes.js file just taking the current tag & figuring it out.